examples/complete/config/ue2-example.yaml

```diff
@@ -0,0 +1,18 @@
+projects:
+  globals:
+    stage: example
+    environment: ue2
+    account_number: "012345678901"
+
+  terraform:
+    vpc:
```

Comment:
We should probably depend on easier examples. Just very simple modules that don't even need to provision infrastructure.

Reply:
Hey @osterman , I'm not sure if I follow your comment exactly. The configuration here should map to projects within the `projects/` folder -- what sort of projects/modules were you thinking of and can you provide an example?

Reply:
`examples/complete/projects/example1/main.tf`
```
output "foobar" {
  value = "hello world!"
}
```

`examples/complete/projects/example2/main.tf`
```
output "foobar" {
  value = "hello world!"
}
```

aka - not a complete VPC module implementation

Reply:
hey @osterman , I had to fix a couple things, but I've got the new examples added if you want to take a look. I'm going to try and knock out the documentation really quick as well 👍